Subject: [PATCH 2/7] GeomPlate_BuildAveragePlane: BasePlan: Don't set yvector to zero. See Following:
From: blobfish <blobfish@gmx.com>

Date: Tue, 29 Sep 2020 06:36:13 -0400

    When we return, the yvector is crossed with x and we crash.
    The z vector is passed in and we calculate the x vector, so just cross those 2 in this case to get y.

---
 src/GeomPlate/GeomPlate_BuildAveragePlane.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/GeomPlate/GeomPlate_BuildAveragePlane.cxx
+++ b/src/GeomPlate/GeomPlate_BuildAveragePlane.cxx
@@ -436,7 +436,7 @@
           || ((Abs(n2)<=myTol)&&(Abs(n3)<=myTol))
           || ((Abs(n1)<=myTol)&&(Abs(n3)<=myTol))) {
       myOX.SetCoord(V3(1),V3(2),V3(3));
-      myOY.SetCoord(0,0,0);
+      myOY = OZ ^ myOX;
     }
     else {
       myOX.SetCoord(V3(1),V3(2),V3(3));
